iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.kitchen.iChef.Controller;
+
+import com.kitchen.iChef.Domain.AppUser;
+import com.kitchen.iChef.Service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    UserService userService;
+
+    @GetMapping(value = "/hello")
+    public String test() {
+        return "hello";
+    }
+
+    @GetMapping
+    public List<AppUser> getAllUsers() {
```

Comment:
We don't want to expose a list of all out users

Reply:
added comment for future purposes